gui/default/syncthing/folder/editIgnoresModalView.html

```diff
@@ -14,12 +14,13 @@
     </dl>
   </div>
   <div class="modal-footer">
-    <div class="pull-left"><span translate>Editing</span> <code>{{currentFolder.path}}{{system.pathSeparator}}.stignore</code></div>
+    <div class="pull-left" ng-show="!editingExisting"><span translate>Creating ignore patterns at</span> <code>{{currentFolder.path}}{{!currentFolder.path.endsWith(system.pathSeparator) ? system.pathSeparator : "" }}.stignore</code> <span translate>(overwrites existing file).</span></div>
```

Comment:
Thanks. This might change if it is decided to append instead of ignore, I will fix it then.